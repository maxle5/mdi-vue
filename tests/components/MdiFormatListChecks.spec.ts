
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiFormatListChecks from "../../src/components/MdiFormatListChecks.vue";

test("MdiFormatListChecks snapshot", () => {
  const wrapper = mount(MdiFormatListChecks, {});
  expect(wrapper.html()).toMatchSnapshot();
});
