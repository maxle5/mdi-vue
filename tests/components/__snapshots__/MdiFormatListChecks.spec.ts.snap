// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiFormatListChecks snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M3,5H9V11H3V5M5,7V9H7V7H5M11,7H21V9H11V7M11,15H21V17H11V15M5,20L1.5,16.5L2.91,15.09L5,17.17L9.59,12.59L11,14L5,20Z" style="fill: currentColor;"></path>
</svg>"
`;
